moto/core/__init__.py

```diff
@@ -1,10 +1,20 @@
-from .models import DEFAULT_ACCOUNT_ID  # noqa
-from .base_backend import BaseBackend, BackendDict, SERVICE_BACKEND  # noqa
-from .common_models import BaseModel  # noqa
-from .common_models import CloudFormationModel, CloudWatchMetricProvider  # noqa
-from .models import patch_client, patch_resource  # noqa
+from .base_backend import SERVICE_BACKEND, BackendDict, BaseBackend
+from .common_models import BaseModel, CloudFormationModel, CloudWatchMetricProvider
+from .models import DEFAULT_ACCOUNT_ID, patch_client, patch_resource
 from .responses import ActionAuthenticatorMixin
 
 set_initial_no_auth_action_count = (
     ActionAuthenticatorMixin.set_initial_no_auth_action_count
 )
+
+__all__ = [
```

Comment:
doing this clears about the need for the noqa pragmas. Also, mypy complains if you import these elsewhere if the `--no-implicit-reexport` (or `--strict`) flag is set, which was why I did it. I don't think you're using `__all__` declarations anywhere in moto before this, so let me know if it's something you'd like to avoid.

Reply:
I think I prefer the existing code. I don't see a usecase where `__all__` is actually useful for Moto-users, and I prefer a few `noqa`'s over having to maintain a duplicate list.

Reply:
fixed.